utils/src/ooinstall/oo_config.py

```diff
@@ -176,6 +177,10 @@ def _read_config(self):
                 if 'Description' in self.settings:
```

Comment:
At some point we need to think about removing support for older versions. If we've supported v1 since at least 3.1, then I would say we can error out and point to the docs if the version field isn't present.